diff -Nur libpng-1.6.37/config.sub libpng-1.6.37-meetix/config.sub
--- libpng-1.6.37/config.sub	2019-04-14 20:10:32.000000000 +0200
+++ libpng-1.6.37-meetix/config.sub	2021-10-22 17:18:37.155948027 +0200
@@ -1366,7 +1366,7 @@
 	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
-	      | -sym* | -kopensolaris* | -plan9* \
+	      | -sym* | -kopensolaris* | -plan9* | -meetix* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
 	      | -aos* | -aros* | -cloudabi* | -sortix* \
 	      | -nindy* | -vxsim* | -vxworks* | -ebmon* | -hms* | -mvs* \
