diff -Nur freetype-2.11.0/builds/unix/config.sub freetype-2.11.0-meetix/builds/unix/config.sub
--- freetype-2.11.0/builds/unix/config.sub	2021-07-19 18:08:21.000000000 +0200
+++ freetype-2.11.0-meetix/builds/unix/config.sub	2021-10-22 17:22:11.252288762 +0200
@@ -1713,7 +1713,7 @@
 	     | hpux* | unos* | osf* | luna* | dgux* | auroraux* | solaris* \
 	     | sym* |  plan9* | psp* | sim* | xray* | os68k* | v88r* \
 	     | hiux* | abug | nacl* | netware* | windows* \
-	     | os9* | macos* | osx* | ios* \
+	     | os9* | macos* | osx* | ios* | meetix* \
 	     | mpw* | magic* | mmixware* | mon960* | lnews* \
 	     | amigaos* | amigados* | msdos* | newsos* | unicos* | aof* \
 	     | aos* | aros* | cloudabi* | sortix* | twizzler* \
