diff -Nur cairo-1.16.0/build/config.sub cairo-1.16.0-meetix/build/config.sub
--- cairo-1.16.0/build/config.sub	2018-08-17 03:11:52.000000000 +0200
+++ cairo-1.16.0-meetix/build/config.sub	2021-10-25 08:47:48.335480653 +0200
@@ -1366,7 +1366,7 @@
 	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
-	      | -sym* | -kopensolaris* | -plan9* \
+	      | -sym* | -kopensolaris* | -plan9* | -meetix* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
 	      | -aos* | -aros* | -cloudabi* | -sortix* \
 	      | -nindy* | -vxsim* | -vxworks* | -ebmon* | -hms* | -mvs* \
diff -Nur cairo-1.16.0/test/pdiff/pdiff.h cairo-1.16.0-meetix/test/pdiff/pdiff.h
--- cairo-1.16.0/test/pdiff/pdiff.h	2018-08-17 03:10:53.000000000 +0200
+++ cairo-1.16.0-meetix/test/pdiff/pdiff.h	2021-10-25 08:46:49.663045806 +0200
@@ -18,14 +18,7 @@
 #define _PDIFF_H
 
 #include <cairo.h>
-
-typedef int bool;
-#ifndef true
-#define true 1
-#endif
-#ifndef false
-#define false 0
-#endif
+#include <stdbool.h>
 
 /* Image comparison metric using Yee's method (and a cairo interface)
  * References: A Perceptual Metric for Production Testing, Hector Yee, Journal of Graphics Tools 2004
